cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -22,6 +22,7 @@
 
 #include "velox/common/memory/MallocAllocator.h"
 #include "velox/common/memory/MemoryPool.h"
+#include "velox/common/memory/SharedArbitrator.h"
 #include "velox/exec/MemoryReclaimer.h"
```

Comment:
Why we need to include `SharedArbitrator.h`? Thought we should avoid this kind of dependency.

Reply:
I just want reuse `velox::memory::SharedArbitrator::ExtraConfig`, would you give some input?